tgui/packages/tgui/interfaces/Terminal.js

```diff
@@ -0,0 +1,17 @@
+import { useBackend } from '../backend';
+import { NoticeBox } from '../components';
+import { Window } from '../layouts';
+
+export const Terminal = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window theme={data.tguitheme} title="Terminal" width={600} height={600}>
+      <Window.Content scrollable>
+        <NoticeBox textAlign="left">
+          {data.uppertext}
+        </NoticeBox>
+        {data.text}
```

Comment:
Consider putting this into a section or a box so that you can add some padding to the edges of the element holding the text. Otherwise it'll make the message look a bit cramped.

Reply:
It would be good to actually wrap it with a <Section>

Reply:
![image](https://user-images.githubusercontent.com/66640614/167547821-f0b4532e-f695-4690-9135-c90fa3b2b715.png)

It definitely looks a lot better